src/tests/cxxfrontend.c

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
Creating the folder `src/tests` might lead to confusions for newcomers.

Reply:
I wasn't sure the best place for this, as the main source folder doesn't seem right.